### ISSUE TITLE
fix: incorrect error of 'missing line wrapping'

### ISSUE THREAD

**Issue by linmeii** [schoero/eslint-plugin-readable-tailwind#10]
I've recently encountered a problem with eslint on a readable-tailwind/multiline rule, where it displays a warning when the classes look correctly formatted.

The warning reads as follows:
```
Missing line wrapping: "
    relative isolate inline-flex items-center justify-center gap-x-2 rounded-lg border text-sm
    font-semibold
  ". eslint(readable-tailwind/multiline)
```

The code where the warning is located:
```ts
export const CustomButtonVariants = tv({
  base: `
    relative isolate inline-flex items-center justify-center gap-x-2 rounded-lg border text-sm
    font-semibold
  `,
});
```

I'm using the tailwind-variants library instead of class-variance-authority, so I set up a list of callees following the documentation as below:
```cjs
'readable-tailwind/multiline': [
  'warn',
  {
    'printWidth': 100,
    'group': 'emptyLine',
    'callees': [
      [
        "tv\\(([^)]*)\\)",
        "[\"'`]([^\"'`]*).*?[\"'`]"
      ]
    ]
  }
],
```

I will be happy to help solve the problem

**Comment by schoero** [comment]
Thank you for reporting.

What version are you using?
You can check with

```sh
npm ls eslint-plugin-readable-tailwind
```

I can't reproduce any false positives using the configuration you have provided and the latest version 1.2.2 of this plugin with the JSX parser. Are you using a different parser?

**Comment by linmeii** [comment]
I am using version 1.2.2
My eslint parser settings:
```cjs
parser: '@typescript-eslint/parser',
parserOptions: {
  ecmaFeatures: {
    jsx: true,
  },
  ecmaVersion: 'latest',
},
```
I've found a potential problem that lies in visual studio code EOL settings.
If EOL is set to CRLF then the plugin displays the warning.
If set to LF then no warning is displayed, so working correctly

**Comment by schoero** [comment]
Thank you for the hint, I can reproduce it now.

The plugin [uses `\n`](https://github.com/schoero/eslint-plugin-readable-tailwind/blob/742faa24d9028791e1576fd9a3757f7058790695/src/rules/tailwind-multiline.ts#L531) to produce the formatted string. This then differs from the original string that uses `\r\n`.

I think the best way to address this is to add a `lineBreak` option, similar to https://eslint.style/rules/js/linebreak-style.

I will try to create a fix by tomorrow.

**Comment by schoero** [comment]
I have [created a pull request](https://github.com/schoero/eslint-plugin-readable-tailwind/pull/11) that fixes this issue.

You can now set an additional `lineBreakStyle` to change the line endings:

```ts
'readable-tailwind/multiline': [
  'warn',
  {
    'printWidth': 100,
    'group': 'emptyLine',
    'lineBreakStyle': 'windows',
    'callees': [
      [
        "tv\\(([^)]*)\\)",
        "[\"'`]([^\"'`]*).*?[\"'`]"
      ]
    ]
  }
],
```

You can install the beta version using

```sh
npm i eslint-plugin-readable-tailwind@beta
```

Please let me know if this works for you.

**Comment by linmeii** [comment]
Can confirm, works like a charm

**Comment by schoero** [comment]
The new version containing this fix is now released on [npm](https://www.npmjs.com/package/eslint-plugin-readable-tailwind?activeTab=readme).
